themes/custom/droopler_theme/scss/components/_geysir_overrides.scss

```diff
@@ -61,12 +64,21 @@
   .paragraph-form .form-actions {
     margin-bottom: 1rem;
   }
+  .paragraphs-collapsed-description:after {
+    background: $color-paris;
+    background: linear-gradient(to right, rgba(255,255,255,0) 0%, $color-paris 100%);
```

Comment:
zamiast rgba(255,255,255,0) to można też z użyciem zmiennej rgba($white,0)
swoją drogą nie wiem jaka jest nadal koncepcja ale zazwyczaj było tak, że kolory takie jak $color-paris
były pierw przypisywane do innych zmiennych jak np.
$geysir-description-background: $color-paris !default;